src/clockwork/amqp.clj

```diff
@@ -30,4 +41,19 @@
     (log/info (format "[amqp/connect] [%s]" (config/amqp-uri)))
     (declare-exchange channel exchange-cfg)
     (declare-queue channel exchange-cfg queue-cfg (keys handlers))
-    (lc/blocking-subscribe channel (:name queue-cfg) (partial message-router handlers))))
+    (lc/blocking-subscribe channel (:name queue-cfg) (partial message-router handlers))
+    channel))
+
+(defn publish-msg
+  [routing-key msg]
+  (try+
+    (let [timeNow (new java.util.Date)]
+      (lb/publish (channel)
+                  (config/exchange-name)
+                  routing-key
+                  (cheshire/encode {:message      msg
+                                    :timestamp_ms (.getTime timeNow)})
+                  {:content-type "application/json"
+                   :timestamp    timeNow}))
```

Comment:
Eventually, **all** of our AMQP messages should have timestamps. At this time, many of our services do not place timestamps in the AMQP messages.

The RabbitMQ timestamps are limited, since they are seconds since epoch. Having another timestamp which is ms since epoch is definitely preferred.

At this time, there is no need for infosquito to have the extra information. 
After writing this, I am going to update the PR s.t. there is a message stating it was sent by clockwork.